uint64 timestamp

uint8 command
uint8 release_hook
uint8 comm_mode
uint8 led_control
uint8 rope_cut
uint8 fuse_check
uint8 clear_rope_length
uint8 clear_weight

int16 fixed_rope_up_length
int16 fixed_rope_down_length

uint32 calibration_factor
uint8 target_address
uint8 priority
